metadata-ingestion/src/datahub/ingestion/source/ge_profiling_config.py

```diff
@@ -84,6 +84,11 @@ class GEProfilingConfig(ConfigModel):
         description="A positive integer that specifies the maximum number of columns to profile for any table. `None` implies all columns. The cost of profiling goes up significantly as the number of columns to profile goes up.",
     )
 
+    profile_if_updated_since_days: Optional[pydantic.PositiveFloat] = Field(
```

Comment:
Please add a note in `updating-datahub.md` file as this is a change in default behaviour.